### PR TITLE
HPCC-14167 Improve docs for -I parameter in ecl cli

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@afishbeck Please review

### CODE REVIEW

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 1]
```diff
@@ -840,11 +842,13 @@ ecl unpublish roxie "FindpersonService*"
         <para><emphasis role="bold">Examples</emphasis>:</para>
 
         <programlisting>ecl run thor findperson.ecl --name=findperson --input=data.xml --wait=1000
+ecl run thor findperson.ecl -I C:\MyECL\
 ecl run thor W20150914-125557 --input=data.xml --wait=1000
 ecl run thor findperson --input=data.xml --wait=1000
 ecl run thor libW20150914-125557.so --input=data.xml --wait=1000
 ecl run thor - --input=data.xml --wait=1000
 ecl run thor findperson.ecl --input="&lt;request&gt;&lt;LName&gt;JONES&lt;/LName&gt;&lt;/request&gt;" 
+ecl run thor findperson.ecl -I C:\MyECL\
```
Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: looks like this same example is listed twice.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 2]
```diff
@@ -840,11 +842,13 @@ ecl unpublish roxie "FindpersonService*"
         <para><emphasis role="bold">Examples</emphasis>:</para>
 
         <programlisting>ecl run thor findperson.ecl --name=findperson --input=data.xml --wait=1000
+ecl run thor findperson.ecl -I C:\MyECL\
```
Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: Have you tested this?  It uses common command processing with eclcc and at least originally didn't support a space after the -I.  It follows gcc style for includes.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 3]
```diff
@@ -1017,14 +1022,14 @@ ecl run thor findperson.ecl --input="&lt;request&gt;&lt;LName&gt;JONES&lt;/LName
                 </row>
 
                 <row>
-                  <entry>-Ipath</entry>
+                  <entry>-I &lt;path&gt;</entry>
```
Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: again, only ok if you've verified the space after -I works.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 4]
```diff
@@ -1017,14 +1022,14 @@ ecl run thor findperson.ecl --input="&lt;request&gt;&lt;LName&gt;JONES&lt;/LName
...
 
-                  <entry>Add path to locations to search for ecl
-                  imports</entry>
+                  <entry>Add path to locations to search for ECL imports
+                  (e.g., -I C:\MyECL\ ) </entry>
```
Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: again, verify space.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml [hunk 5]
```diff
@@ -1017,14 +1022,14 @@ ecl run thor findperson.ecl --input="&lt;request&gt;&lt;LName&gt;JONES&lt;/LName
...
                 </row>
 
                 <row>
-                  <entry>-Lpath</entry>
+                  <entry>-L &lt;path&gt;</entry>
```
Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: Same space question for -L

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 2 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: I verified that it works with a space.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 3 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: I verified that it works with a space.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 4 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: I verified that it works with a space.

File: docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml (hunk 5 above)
Comment (JamesDeFabia) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: I did not verify for Libraries

Comment (afishbeck) [inline docs/HPCCClientTools/CT_Mods/CT_ECL_CLI.xml]: should be consistent.  It's processed the same way

**Comment by afishbeck** [comment]
@JamesDeFabia looks good

**Comment by JamesDeFabia** [comment]
@afishbeck Please look again

**Comment by afishbeck** [comment]
@JamesDeFabia looks good

**Pull request merged**